go.mod

```diff
@@ -0,0 +1,10 @@
+module example.com/m/v2
```

Comment:
please use `go mod init github.com/harness-community/drone-archive` to generate mod and sum files

Reply:
Done